packages/ui/src/tailwind.config.ts

```diff
@@ -183,6 +183,9 @@ const config = {
         '*:not(svg, svg *), use': {
           all: 'revert',
         },
+        ':root': {
+          colorScheme: 'light dark',
+        },
```

Comment:
If we are not shipping the dark one, it will be a breaking change as it's already supported in the current components

Reply:
> Hmm forgive me if I'm wrong but aren't we only shipping the light UI for initial launch?

Correct, this should not have an effect on that though since we're only toggling the `light`/`dark` vars within theme-builder rendering. In the future this will move to the individual components via the appearance prop.

Let me know if I am missing something obvious though, I toggle my system theme and it stays in light mode, it only updates to dark theme when toggling the theme builder toggle.

Reply:
> If we are not shipping the dark one, it will be a breaking change as it's already supported in the current components

This is just to get this onto dashboard to start dogfooding.